### ISSUE TITLE
Update dependency: jtskit --> jsontableschema

### ISSUE THREAD

**Issue by brew** [okfn/goodtables#61]
Once jsontableschema 0.5.0 is available, the jtskit dependency should be changed to jsontableschema.

Some hints where it's used and defined in goodtables: https://github.com/okfn/goodtables/search?utf8=%E2%9C%93&q=jtskit

**Comment by augusto-herrmann** [comment]
Goodtables v0.5.6 is unusable because of this bug.

Steps to reproduce:

```
$ pip install goodtables

...

$ goodtables
...
  File "... /local/lib/python2.7/site-packages/goodtables/processors/schema.py", line 7, in <module>
    import jtskit
ImportError: No module named jtskit
```

Attempting to install jtskit manually does not solve it:

```
$ pip install jtskit
Requirement already satisfied (use --upgrade to upgrade): jtskit in ./pyenv/lib/python2.7/site-packages
Requirement already satisfied (use --upgrade to upgrade): jsontableschema>=0.5.1 in ./pyenv/lib/python2.7/site-packages (from jtskit)
Cleaning up...
```

I suppose the only solution, for now, is probably using goodtables from the latest commit from the repository. Simply using stable versions with pip is not currently possible.

**Comment by brew** [comment]
@pwalsh Does this just need a new release of Goodtables to pypi?

**Comment by pwalsh** [comment]
@brew not sure and can't check now. *if* you have time to check (and push) would be great.

**Comment by augusto-herrmann** [comment]
I think so. When I installed it last night by using `git clone` (from master) and then `pip install -U .`, the CLI worked.

**Comment by pwalsh** [comment]
Fixed in `0.6.0` https://github.com/okfn/goodtables/tree/0.6.0
